napari/layers/labels/labels.py

```diff
@@ -937,10 +943,77 @@ def _partial_labels_refresh(self):
         colors_sliced = self._raw_to_displayed(
             raw_displayed, data_slice=updated_slice
         )
+        self._slice.image.view[updated_slice] = colors_sliced
```

Comment:
I do not know why I need to this line. On main the `view` array is updated, but I cannot find where. But After I add this the test are passing.

Reply:
@Czaki which tests? This of course reminds me of #6112, and indeed [this comment](https://github.com/napari/napari/pull/6112/files#r1291613760) by @andy-sweet:

> If needed, I think you could fix this fairly easily by doing something like `self._slice.image.view[displayed_indices] = self._raw_to_displayed(self._slice.image.raw[displayed_indices])` (or maybe just `:` instead of `displayed_indices` depending on how `_raw_to_displayed` works). Either here or in `_partial_labels_refresh`.

So I guess it *was* needed, 😅 but it would be good to keep a note about it here, given #6185. So:

```suggestion
        # This line is needed for test_... to pass
        # See https://github.com/napari/napari/pull/6112/files#r1291613760
        # and https://github.com/napari/napari/pull/6185
        self._slice.image.view[updated_slice] = colors_sliced
```

Reply:
@Czaki 👆?